generators/chipyard/src/main/scala/example/GCD.scala

```diff
@@ -46,10 +46,11 @@ trait HasGCDIO extends BaseModule {
 }
 
 // DOC include start: GCD blackbox
-class GCDMMIOBlackBox(val w: Int) extends BlackBox(Map("WIDTH" -> IntParam(w))) with HasBlackBoxResource
+class GCDMMIOBlackBox(val w: Int) extends BlackBox(Map("WIDTH" -> IntParam(w))) with HasBlackBoxPath
```

Comment:
Were you intending to change this example?

Reply:
This is to expose the bug. I'll revert it once a solution is found (see the comment below).

Reply:
Reverted